test/classes/deinitializers/deinitAndDestruct.good

```diff
@@ -0,0 +1,3 @@
+deinitAndDestruct.chpl:11: error: ambiguous call 'C.deinit()'
+deinitAndDestruct.chpl:2: note: candidates are: C.~C()
+deinitAndDestruct.chpl:6: note:                 C.deinit()
```

Comment:
You're probably asking Vass, but my opinion is that we should take the path of least resistance in this case.